std/range/package.d

```diff
@@ -3881,9 +3881,12 @@ pure @safe nothrow @nogc unittest
 }
 
 /++
-    Convenience function which calls
+    `drop` is a convenience function which calls
     $(REF popFrontN, std, range, primitives)`(range, n)` and returns `range`.
-    `drop` makes it easier to pop elements from a range
+    Unlike `popFrontN`, the range argument is not passed by `ref`, so
+    it may not be mutated.
```

Comment:
Maybe we should also outline the condition that determines when this “may” may apply.